components/AdminNav/AdminNav.tsx

```diff
@@ -1,6 +1,7 @@
 // Copyright (c) Gridiron Survivor.
 // Licensed under the MIT License.
 
+'use client';
```

Comment:
Why is this changed to a client component?

Reply:
it works without it. I thought it would need it because of how the test is using auth, but I took it out